File: voicedrawing/lib_common_voyah/src/main/java/com/lib/common/voyah/service/IVoiceService.java
package com.lib.common.voyah.service;


import android.content.Context;
import android.content.Intent;

/**
 * author : jie wang
 * date : 2025/3/18 11:32
 * description :
 */
public interface IVoiceService {

    void postBurialPointEvent(String eventType, String triggerMode, long time, String extraStr);

    int getDisplayId();

    void sendBroadcastToGallery(Context context, Intent intent);

    String getWakeUpWord();

    Context getDisplayContext(Context context, int displayId);
}

File: voiceui/lib_common_voyah/src/main/java/com/lib/common/voyah/service/IVoiceService.kt
package com.lib.common.voyah.service

/**
 *  author : jie wang
 *  date : 2025/4/24 17:00
 *  description :
 */
interface IVoiceService {
    fun getWaveWindowTag(
        location: Int,
        passengerState: Boolean,
        ceilingState: Boolean
    ): String

    fun getWaveX(
        location: Int,
        windowMargin: Int,
        passengerState: Boolean,
        ceilingState: Boolean,
        screenWidth: Int,
        windowWidth: Int
    ): Int

    fun hasMultipleDisplay(): Boolean
}